src/python/grpcio_tests/tests/unit/_channel_connectivity_test.py

```diff
@@ -32,12 +32,12 @@
 import threading
 import time
 import unittest
-from concurrent import futures
 
 import grpc
 from grpc import _channel
 from grpc import _server
 from tests.unit.framework.common import test_constants
+from tests.unit._thread_pool import RecordingThreadPool
```

Comment:
[Import only packages and modules (mostly modules), not classes, functions, or other elements of modules.](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports)